parser.go

```diff
@@ -318,7 +318,6 @@ func eatItem(item string) {
 	if val, ok := inventory[item]; ok {
 		if val.IsEdible {
 			fmt.Println("That was delicious! Your strength has been restored.")
-			delete(inventory, item)
```

Comment:
I'm not on board with this. If you eat an edible item, it's gone forever. (Or until your parents go to the store and get more.)

Reply:
But you will lose automatically if you have eaten the corn flakes, which you need to fix the shrink ray and are also the only edible thing left, and the player has no way of knowing this.

Reply:
I changed the message that's printed.